homeassistant/components/deconz/climate.py

```diff
@@ -172,6 +173,21 @@ async def async_set_hvac_mode(self, hvac_mode: HVACMode) -> None:
                 mode=HVAC_MODE_TO_DECONZ[hvac_mode],
             )
 
+    @property
+    def hvac_action(self) -> str | None:
```

Comment:
The return value should just be `HVACAction`.

Reply:
Thanks! Will fix.

Reply:
Resolved with https://github.com/home-assistant/core/pull/79485